packages/compass-indexes/src/components/search-indexes-modals/base-search-index-modal.tsx

```diff
@@ -40,6 +44,22 @@ const bodyStyles = css({
   gap: spacing[3],
 });
 
+const templateToolbarStyles = css({
+  display: 'flex',
+  flexDirection: 'row',
+  gap: spacing[3],
+  overflow: 'hidden',
```

Comment:
Because of the overflow hidden this selector gets cut off when switching between the options (and even when nothing is selected URL is not visible):

![image](https://github.com/mongodb-js/compass/assets/5036933/9473ea8d-71c5-4ab2-88a5-7fee2fa1e065)

Reply:
Removed the overflow hidden and tested different sizes, now it should not get cut off. Thanks for pointing out!

Reply:
Did you revert this change or something went wrong with the merge from main? The hidden is still here a still causes issues:

<img width="578" alt="image" src="https://github.com/mongodb-js/compass/assets/5036933/a5d44173-1b7b-491c-b32f-c6898ac4e46d">

Reply:
I think I did something wrong merging with main, I'm merging again (more changes 😄) and I'll remove the hidden.